net-misc/putty/patches/putty-0.60.patch

```diff
@@ -1,24 +0,0 @@
-diff -urN putty-0.60/unix/configure.ac putty-0.60-haiku/unix/configure.ac
---- putty-0.60/unix/configure.ac	2009-02-07 20:49:11.000000000 +0000
-+++ putty-0.60-haiku/unix/configure.ac	2009-02-07 21:50:05.000000000 +0000
-@@ -14,14 +14,16 @@
-     CFLAGS="$CFLAGS -Wall -Werror"
- fi
- 
--AC_CHECK_HEADERS([utmpx.h sys/select.h],,,[
--#include <sys/types.h>
--#include <utmp.h>])
-+AC_CHECK_HEADERS([utmpx.h])
-+AC_CHECK_HEADERS([sys/select.h])
-+AC_CHECK_HEADERS([sys/types.h])
-+AC_CHECK_HEADERS([utmp.h]) 
- 
--AM_PATH_GTK([1.2.0], [all_targets="all-cli all-gtk"], [all_targets="all-cli"])
-+#AM_PATH_GTK([1.2.0], [all_targets="all-cli all-gtk"], [all_targets="all-cli"])
- AC_SUBST([all_targets])
- 
- AC_SEARCH_LIBS([socket], [xnet])
```

Comment:
Why remove the other check, wouldn't that break it on other platforms?  Goal should be to write patches such that they could be upstreamed, so that we don't have to repatch for every new release.